tests/test_hypersphere.py

```diff
@@ -420,7 +420,9 @@ def test_exp_and_dist_and_projection_to_tangent_space(self):
         expected = gs.linalg.norm(tangent_vec) % (2 * gs.pi)
         self.assertAllClose(result, expected)
 
+    @geomstats.tests.np_and_pytorch_only
     def test_exp_and_dist_and_projection_to_tangent_space_vec(self):
+        # TODO(ninamiolane): Fix hypersphere's exp with assignment
```

Comment:
@pchauchat

Reply:
Found the problem, see comment https://github.com/geomstats/geomstats/pull/612/files#r413596996

Reply:
Yess, it works now! @pchauchat you're the GOAT!